test/test_transformers.py

```diff
@@ -2469,6 +2469,73 @@ def test_cudnn_attention_different_dk_dv(self, device):
 
         self.assertEqual(actual.contiguous(), math_ref.contiguous().to(dtype), atol=1e-3, rtol=1e-2)
 
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FUSED_ATTENTION, "Fused SDPA was not built for this system")
+    @parametrize("backend", PLATFORM_SPECIFIC_SDPA, name_fn=lambda x: x.name)
+    @parametrize("compile_mode", ["eager", "inductor"])
```

Comment:
So locally all test pass but that doesn't seem possible, since i didnt even update the meta for mem eff yet

i wonder if there are too many recompiles and then just falls back to eager..